docs/source/platform/schema-validation.md

```diff
@@ -1,60 +1,101 @@
+- **Removal**: removal of schema element that consumer use, such as type, field, enum, argument, or other
 
-It might be tempting to version a GraphQL API the same way, but it's unnecessary with the right techniques.  By following the strategies and precautions outlined in this guide and using Apollo tooling that adds clarity to every change, many iterations of an API can be served from a single endpoint.
+Each of these overall types contains different changes codes, such as `TYPE_REMOVED`, `ARG_DEFAULT_VALUE_CHANGE`, `ENUM_VALUE_ADDED`, or `FIELD_CHANGED_KIND`, which are displayed in the cli output.
 
-<h3 id="field-usage">Field usage</h3>
+<h3 id="metrics">Assigning severity</h3>
 
-Rather than returning extensive amounts of data which might not be necessary, GraphQL allows consumers to specify exactly what data they need.  This field-based granularity is valuable and avoids "over-fetching" but also makes it more difficult to understand what data is currently being used.
+When metrics are sent to a service, the changes are verified in the following manner:
 
-To improve the understanding of field usage within an API, Apollo Server extends GraphQL with rich tracing data that demonstrates _how_ a GraphQL field is used and _when_ it's safe to change or eliminate a field.
+- **Addition**: Automatically marked as a `Notice`
+- **Update**: Automatically marked as a `Warning`
+- **Removal**: If an operation uses the removed type, field, enum, argument, or other schema element, marked as `Failure`
 
-> For details on how tracing data can be used to avoid shipping breaking changes to clients, check out the schema history tooling in [Apollo Engine](https://www.apollographql.com/platform) which utilizes actual usage data to provide warnings and notices about changes that might break existing clients.
+The overall schema validation check will fail if there are any changes marked as a `Failure`.
 
-Since GraphQL clients only receive exactly what they ask for, adding new fields, arguments, queries, or mutations won't introduce any new breaking changes and these changes can be confidently made without consideration about existing clients or field usage metrics.
+> Occasionally a service or schema tag does not contain any metrics. In this case, schema validation does cannot verify changes, so removals are automatically set as `Failure`. Additionally warnings will fail Github status checks.
+
+<h2 id="performing-changes">Strategies for performing schema changes</h2>
+
+Defining strategies to perform these changes with minimal impact on clients enables maintainable evolving a schema in response to rapidly changing product requirements. The insight necessary for these techniques is adding new fields, arguments, queries, or mutations won't introduce any new breaking changes. These changes can be confidently made without consideration about existing clients or field usage metrics, since GraphQL clients receive exactly what they ask for.
```

Comment:
typo: maintainable -> maintainably

Reply:
We should probably write specific guidance on _rather than updating a field in place, we strongly recommend deprecating the old field and creating a new one instead, in case there's any concern that modifying the field in place could break current clients_.